pan/types.pan

```diff
@@ -1002,3 +1002,23 @@ type cpu_architecture = string with match (SELF, '^(i386|ia64|x86_64|sparc|aarch
     desc = Linux capabilities, see CAPABILITIES(7)
 }
 type linux_capability = string with match(SELF, '^CAP_[A-Z_]+$');
+
+@documentation{
+    desc = Search path, a colon seperated set of absolute paths which may include . as a reference to current working directory
+}
+type string_search_path = string_trimmed with {
+    if (! match(SELF, '^[^:]+?(?::[^:]+)+$')) {
+        error('The string "%s" does not look like a search path.', SELF);
```

Comment:
The error string should include the regex that's being matched against. Otherwise I have to find the source code to figure out why %s doesn't match.

    error(The string "%s" does not look like a search path, i.e. must match /^[^:]+?(?::[^:]+)+$/', SELF);

Reply:
Good point!